libs/api/features/farewell/src/lib/be-farewell.service.ts

```diff
@@ -30,7 +50,8 @@ export class BeFarewellService {
 
     return farewell;
   }
-  async createFarewell(farewell: Ifarewell) {
+
+  async createFarewell(farewell: IFarewell) {
```

Comment:
I guess since we can have farewell for kudo we also have to have 

```
 kudoBoardId: farewell.kudoBoardId
          ? new mongoose.Types.ObjectId(farewell.kudoBoardId)
          : null,
```